symtabAPI/h/Symtab.h

```diff
@@ -235,6 +235,8 @@ class SYMTAB_EXPORT Symtab : public LookupInterface,
 
    /*****Query Functions*****/
    bool isExec() const;
+   bool isExecutable() const;
+   bool isSharedLibrary() const;
```

Comment:
Hmm... This is an API break. I know we discussed this before, but what was our rationale for removing these?

Reply:
These are added methods.  Adding two non-virtual functions shouldn't be an API or ABI break.

Reply:
Wow. I must be going color blind because I thought those were red...